src/modules/flow/json/json.c

```diff
@@ -992,6 +993,7 @@ json_path_add_new_element(struct sol_flow_node *node, struct json_element *base_
             SOL_INT_CHECK(r, == -ENOMEM, r);
             SOL_INT_CHECK_GOTO(r, < 0, error_parse);
 
+            break;
```

Comment:
The reason to not having that break is the reason @edersondisouza commented. It is not an error. it is intended. I remember having tests that explores that, and they are suppose to fail after your changes. Have you tested that?